x-pack/test/security_solution_cypress/cypress/e2e/ai4dsoc/privileges/security_privileges.cy.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { login } from '../../../tasks/login';
+import { visit } from '../../../tasks/navigation';
+
+const selectSpaces = (): void => {
+  cy.get('[data-test-subj="addSpacePrivilegeButton"]').click();
+  cy.get('[data-test-subj="spaceSelectorComboBox"]').should('be.visible');
+
+  // select space
+  cy.get('[data-test-subj="spaceSelectorComboBox"]').click();
+  cy.get('[data-test-subj="spaceSelectorComboBox"]').type('Default');
```

Comment:
If we are targetting the default space, it would be great if the action itself mentions that.

Reply:
It would be for any space actually.

Reply:
updated in e80ded6b30abdf04dc6ee9e24d71878e3d144f27